src/main/webapp/app/upload/upload.component.ts

```diff
@@ -67,4 +135,15 @@ export class UploadComponent {
         this.error = '';
         this.uploadResponse = { status: '', message: '' };
     }
+
+    private getConflicts() {
+        this.newPool.forEach((value, key) => {
+            if (this.currentPool.has(key) && JSON.stringify(this.currentPool.get(key)) !== JSON.stringify(this.newPool.get(key))) {
+                this.newConflictPool.push(value);
+                this.currentConflictPool.push(value);
```

Comment:
This one is wrong, you are pushing to currentConflictPool a pool from newPool instead of from currentPool. In the end both newConflictPool and currentConflictPool are equal. Also maybe instead of pushing to conflicts just by same id, test if their MaximumCount is different, that way we reduce number of unnecessary conflicts.

Reply:
I am not comparing only by id `JSON.stringify(this.currentPool.get(key)) !== JSON.stringify(this.newPool.get(key))`

Reply:
fixed, I have also changed `JSON.stringify` approach